openforcefield/tests/test_io.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+#=============================================================================================
+# MODULE DOCSTRING
+#=============================================================================================
+
+"""
+Test classes and function in module openforcefield.typing.engines.smirnoff.io.
+
+"""
+
+
+#=============================================================================================
+# GLOBAL IMPORTS
+#=============================================================================================
+
+import ast
+
+import pytest
+from simtk import unit
+
+from openforcefield.typing.engines.smirnoff.io import (
+    _ast_unit_eval,
+    _extract_attached_units,
+)
+
+
+#=============================================================================================
+# QUANTITY PARSING UTILITIES
+#=============================================================================================
+
+@pytest.mark.parametrize('unit_string,expected_unit',[
+    ('kilocalories_per_mole', unit.kilocalories_per_mole),
+    ('kilocalories_per_mole/angstrom**2', unit.kilocalories_per_mole/unit.angstrom**2),
```

Comment:
Glad to see this in the test set. This is one of the trickier things we need to parse.